dist/package.json

```diff
@@ -6,7 +6,7 @@
     },
     "description": "Design tokens for the Sage Design System.",
     "author": "The Sage Group plc",
-    "version": "1.86.0",
+    "version": "0.2.0",
```

Comment:
this shouldn't be overwritten

Reply:
right ;)

Reply:
Actually whole `/dist` folder shouldn't be commited.